Project.toml

```diff
@@ -6,6 +6,8 @@ version = "0.1.0"
 [deps]
 Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 CRC32c = "8bf52ea8-c179-5cab-976a-9e18b702a9bc"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
```

Comment:
Images is a really large dependency as it is basically a metapackages for all of JuliaImages
If we just need `colorview` than I think `ImageCore` is fine